### DISCUSSION TITLE
Simulation crashes whenever Lunar Surface Ops is selected from Demos.

### DISCUSSION THREAD

**Discussion by hayyamiqbal** [SimDynamX/SpaceTeamsPRO#55] [2024 Lunar SAR Competition Q&A]
![WhatsApp Image 2024-10-31 at 15 55 16](https://github.com/user-attachments/assets/526371a4-a25d-4922-8df8-3f60d80d34ce)

Demo doesnt even load before giving this error. Tried this on 2 different devices with 16GB & 64GB of RAM i9. 

Tried with both Personal & team account.

**Comment by connorjak** [comment]
Converted to issue 
- #59

Thanks for the bug report! Unrelated to lunar SAR competition.

**Reply by hayyamiqbal** [reply]
I get the same error when I try to import the example Sim for task 1.

**Reply by connorjak** [reply]
Can you send your Task1 sim logs?
